src/api/swapTypes.ts

```diff
@@ -0,0 +1,50 @@
+import { toBitcoin } from "satoshi-bitcoin-ts";
+import { fromWei } from "web3-utils";
+
+export interface Asset {
+  name: string;
+  quantity: string;
+  token_contract?: string;
+}
+
+export function toMainUnit(asset: Asset) {
+  switch (asset.name) {
+    case "ether":
+      return fromWei(asset.quantity, "ether") + " ETH";
+    case "bitcoin":
+      return toBitcoin(asset.quantity) + " BTC";
+    case "erc20":
+      // TODO: Find and use token symbol if possible
```

Comment:
❓ Did you create a ticket for this?

Reply:
No... I ended up removing almost all of these TODOs because they're out of scope for the time being.